src/tests/networkcontrol/NetworkControl_Assign_001.test.js

```diff
@@ -0,0 +1,41 @@
+import {
+  pluginDeactivate,
+  pluginActivate,
+  assignNetworkAdapter,
+} from '../../commonMethods/commonFunctions'
+import constants from '../../commonMethods/constants'
+
+export default {
+  title: 'NetworkControl - Assign 001',
+  description: 'Check the Assign Functionality of Network Control Module',
+  steps: [
+    {
+      description: 'Check if NetworkControl Plugin is stopped correctly',
+      test: pluginDeactivate,
+      params: constants.networkControlPlugin,
+      assert: 'deactivated',
+    },
+    {
+      description: 'Check if NetworkControl Plugin is started correctly',
+      test: pluginActivate,
+      params: constants.networkControlPlugin,
+      assert: 'activated',
+    },
```

Comment:
Should we really be starting/restating the network plugin prior to each test?

Or could we just test the starting/stopping of the plugin once and run other flush/assign/etc tests without restarting the plugin?

Reply:
This is to make sure the test is independent and no previous test run affects this test. Please let me know if you have better idea to achieve this.